docs/source/algorithms/MaskDetectors-v1.rst

```diff
@@ -309,7 +367,88 @@ Output
   Detector 102 is masked: False
   Detector 103 is masked: True
   Detector 104 is masked: False
+  
+Example 5: Specifying a masking range
+#####################################
+
+.. testcode:: ExMaskInRange
+
+  # Create a data workspace
+  ws = CreateSampleWorkspace()
+  # Mask 3 detectors using the masking range
+  MaskDetectors(ws, StartWorkspaceIndex=2, EndWorkspaceIndex=4)  
 
+  # Check masking of first 6 detectors
+  for ind in xrange(0,6):
+    det = ws.getDetector(ind)
+    print 'Detector',det.getID(),'is masked:',det.isMasked()
```

Comment:
I see. Interesting. So,
```
print('a: ',det.getID())
```
thinks it's a tuple in Python 2.x. One could use string formatting here:
```
print('Detector {} is masked: {}'.format(det.getID(), det.isMasked))
```
to make python 3 and doctests happy.